apps/examples/testcase/ta_tc/task_manager/itc/itc_taskmanager_main.c

```diff
@@ -672,21 +744,15 @@ void itc_taskmanager_group_unregister_p(void)
 	ret = task_manager_start(tm_sample_handle3, TM_RESPONSE_WAIT_INF);
 	TC_ASSERT_EQ_CLEANUP("task_manager_start", ret, OK, unregister_tasks_4(sample_info->tm_gid));
 
-	usleep(100);
+	usleep(1);
 
-	unregister_tasks_4(sample_info->tm_gid);
+	int status;
 
-	TC_SUCCESS_RESULT();
-}
+	waitpid(pid_tm_task9, &status, 0);
 
-void itc_exit_handler(void *info)
-{
-	if (strncmp((char *)info, EXIT_CB_CHECK_STR, strlen(EXIT_CB_CHECK_STR) + 1) == 0) {
-		int ret;
+	unregister_tasks_4(sample_info->tm_gid);
 
-		ret = task_manager_pause(tm_sample_handle, TM_RESPONSE_WAIT_INF);
-		TC_ASSERT_EQ("task_manager_pause", ret, OK);
-	}
+	TC_SUCCESS_RESULT();
 }
 
 void itc_stop_cb(void *data)
```

Comment:
Please rename the function name "as remove itc from supporting function name  and remove assert Macro from supporting function"

Reply:
changes done.